snc.sh

```diff
@@ -72,6 +73,12 @@ if test -z ${OPENSHIFT_INSTALL-}; then
     OPENSHIFT_INSTALL=./openshift-baremetal-install
 fi
 
+if [[ ${CLUSTER_PATCH} == "enabled" ]]
+then
+   echo "Patching cluster KAO and KCMO"
+   patch_operators_and_create_new_release
```

Comment:
please describe more clearly 'which operators' are patched here.

Reply:
There is `echo "Patching cluster KAO and KCMO"` or you mean not using short form for those?

Reply:
Now it's only about creating a new release without doing the patching.